### PR TITLE
Correctly format the notes sections

### PR DESCRIPTION
**Pull request by dwelch-r7**
Nothing much going on here, just found a handful of incorrectly formatted/placed notes while working on displaying them for pro, just tidying them up

I think I got them all but if anyone knows of any missing drop a comment and I'll address that too

### CODE REVIEW

File: modules/exploits/windows/browser/malwarebytes_update_exec.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def initialize(info = {})
       'References'     =>
         [
           [ 'CVE', '2014-4936' ],
-          [' OSVDB', '116050'],
+          ['OSVDB', '116050'],
```
Comment (bcoles) [inline modules/exploits/windows/browser/malwarebytes_update_exec.rb:38]: :thinking:

Comment (gwillcox-r7) [reply modules/exploits/windows/browser/malwarebytes_update_exec.rb:38]: I'd also note this doesn't follow the pre and post spacing that the above entry has, so would be good to follow that and keep things consistent within the same file.

**Review by gwillcox-r7 — CHANGES_REQUESTED** [review]
Looks good minus the one change that was mentioned earlier

File: modules/exploits/windows/browser/malwarebytes_update_exec.rb (hunk 1 above)
Comment (gwillcox-r7) [reply modules/exploits/windows/browser/malwarebytes_update_exec.rb:38]: Fixed in a075c676a6caf7f02a806d7f3adb314fa8130c88

**Pull request merged**